### PR TITLE
Base

### PR DESCRIPTION
**Pull request by Dhiraj240**
Intial tests .
#258 
controllers.go shows undefined Driver.

### CODE REVIEW

**Comment by Dhiraj240** [comment]
@andyzhangx  Needs member access for /ok-to-test  I personally got errors on ```controller.go```
Same thing happened when I used ```version.go``` there was ```undefined driverName``` for which I opened a issue and submitted PR while on tests.
Here the undefined conventions are more, May I know where I am loosing here?

File: pkg/azurefile/controllerserver_test.go [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+				}
+
+				mockCtl := gomock.NewController(t)
+				defer mockCtl.Finish()
+
+				mockCloud := mocks.NewMockCloud(mockCtl)
+				mockCloud.EXPECT().GetDiskByName(gomock.Eq(ctx), gomock.Eq(req.Name), gomock.Eq(stdVolSize)).Return(nil, cloud.ErrNotFound)
+				mockCloud.EXPECT().CreateDisk(gomock.Eq(ctx), gomock.Eq(req.Name), gomock.Eq(diskOptions)).Return(mockDisk, nil)
+
+				awsDriver := controllerService{
```
Comment (andyzhangx) [inline pkg/azurefile/controllerserver_test.go:922]: can you change the name?

**Pull request closed without merging**